src/main/java/com/flightstats/hub/dao/file/SingleContentService.java

```diff
@@ -31,7 +33,7 @@
     public SingleContentService() {
         String contentPath = FileUtil.getContentPath();
         logger.info("using {}", contentPath);
-        fileSpokeStore = new FileSpokeStore(contentPath);
+        fileSpokeStore = new FileSpokeStore(contentPath, HubProperties.getSpokeTtlMinutes(SpokeStore.WRITE));
```

Comment:
I think this should use an injected FileSpokeStore from HubBindings.

Reply:
I agree but since it was like that before I figured it should be addressed in its own issue/PR. I'll add an issue for it.

Reply:
See #971